zqd/zqdtest/zqdtest.go

```diff
@@ -0,0 +1,58 @@
+package zqdtest
```

Comment:
I'm not sure this set of functions really worthy of a separate package. The logic in `NewRequest` and `ExecRequest` seem too small to be separated into functions.

For CreateSpace/WriteSpace: Consider folding the content of WriteSpace into CreateSpace, and then making CreateSpace *not* take a testing.T, and just return an error. That new CreateSpace function could live inside the api package, with a minimal set of (non-test specific) added imports.